### ISSUE TITLE
configuring vim keybindings with `bind -v` breaks tab completion

### ISSUE THREAD

**Issue by benesch** [knz/go-libedit#1]
Naively configuring vim keybindings a la

```
$ cat ~/.editrc
bind -v
```

breaks tab completion and ^C interrupts. Arguably not a bug in go-libedit, but I've only ever configured readline and I expected `bind -v` to work like `set editing-mode vi`. Fixed with a smarter `~/.editrc`, but @knz you mentioned we might be able to do something on the go-libedit side too?

```
$ cat ~/.editrc
bind -v
bind "^R" em-inc-search-prev
bind "^C" rl_interrupt
bind \\t rl_complete
```

**Comment by knz** [comment]
Addressed in 438167ad0c588fe6f2e87cb6ba8a4e9753e6d26e.

**Comment by benesch** [comment]
Thanks, @knz!
